### ISSUE TITLE
Internal links seem to be broken recently

### ISSUE THREAD

**Issue by adriatic** [GitbookIO/gitbook#1351]
I have a book, which uses internal links to facilitate jumps within a long chapter. This worked just fine until today. 

More detals:
1. At the top of the page, I have this anchor: `<a id="core-anchor"></a>`
2. In the mid-section of this same page I have this link: `[Go to core tutorial](#core-anchor)` which is correctly resolved to be `https://aurelia-ui-toolkits.gitbooks.io/kendoui-bridge-docs/content/developers-tutorials/es2016.html#pro-anchor`
3. Click on that link does nothing - however, using the context menu (see below)

![image](https://cloud.githubusercontent.com/assets/2712405/15716763/e1ceb822-27f1-11e6-9c3e-fd95ff8351c6.png)

correctly renders the document at the anchor location.

**Comment by SeaDude** [comment]
Noticed you named the anchor "core-anchor" but in your URL it's resolving to "pro-anchor". Is this the issue here?

**Comment by adriatic** [comment]
No, that is not the issue as this document defines core-anchor and pro-anchor and has links to both

**Comment by SamyPesse** [comment]
It's working for me (on Safari and Google Chrome), it scrolls the page to the right position.

What browser are you using?

**Comment by adriatic** [comment]
It WAS working for me until the day I made the problem report. It IS working for me today as well, so I wanted to check the Chrome version – however, Chrome updated itself before I was able to see the version. Now after the update, the Chrome version is  51.0.2704.79 m

 

Either you updated the GitBook app, or Chrome updated its browser, as I do not believe that I made some pilot error: it steadfastly did nothing when clicked on any of these links, and it worked just if I asked the book to open the document in a different window. 

 

If this ever happens again, I will let you know.

 

Thanks, Samy :)
